collector/container/konflux.Dockerfile

```diff
@@ -1,26 +1,17 @@
 ARG BUILD_DIR=/build
 ARG CMAKE_BUILD_DIR=${BUILD_DIR}/cmake-build
 
-
 # Builder
 # TODO(ROX-20312): we can't pin image tag or digest because currently there's no mechanism to auto-update that.
-# TODO(ROX-20651): use content sets instead of subscription manager for access to RHEL RPMs once available.
-FROM registry.access.redhat.com/ubi8/ubi:latest AS ubi-normal
-FROM ubi-normal AS rpm-implanter-builder
-
-COPY --from=ubi-normal / /mnt
-COPY ./.konflux /tmp/.konflux
+FROM registry.access.redhat.com/ubi8/ubi:latest AS builder
 
-# TODO(ROX-20234): use hermetic builds when installing/updating RPMs becomes hermetic.
-RUN /tmp/.konflux/scripts/subscription-manager-bro.sh register /mnt && \
-    dnf -y --installroot=/mnt upgrade --nobest && \
-    dnf -y --installroot=/mnt install --nobest \
+RUN dnf -y install --nobest \
         make \
         wget \
         unzip \
         clang \
         llvm \
-        cmake \
+        cmake-3.18.2-9.el8 \
```

Comment:
Looks like we are back to pinning the cmake version which was removed in #2027, is this intentional?

Reply:
Sharp eyes! I'm still iterating on the Tom's PR #2013 which is pre #2027. I'm not ready to invite for reviewing yet but thanks for the pointer, I'll deal with it if I ultimately will go ahead with this PR.

Reply:
I believe this one is ready if you'd like to have a look.